Podfile

```diff
@@ -7,7 +7,8 @@ target 'Pokedex_ios' do
 
   # Pods for Pokedex_ios
   pod 'SwiftLint', '0.47.1'
-
+  pod 'Kingfisher'
```

Comment:
don't forget to always add the version on the pods

Reply:
done :)